stable/raccoon/README.md

```diff
@@ -0,0 +1,97 @@
+# Raccoon
+
+A [helm](https://helm.sh/) chart for [Raccoon](https://github.com/odpf/raccoon)
+
+## Introduction
+
+This chart can be used for deploying [Raccoon](https://github.com/odpf/raccoon) on Kubernetes using helm
+
+## Prerequisites
+
+Install [helm3](https://helm.sh/docs/intro/install/#helm)
+
+## Installing the Chart
+
+### Install released version using Helm repository
+
+* Add the odpf charts repo
+  
+```bash
+$ helm repo add odpf https://odpf.github.io/charts/
+```
+
+* Install it with Helm 3
+  
+```bash
+$ helm install my-release odpf/raccoon
+```
+
```

Comment:
Could we also explain what happened with the command, ie., the deployment (CM, Raccoon Pod, Telegraf sidecar etc.,)